ci/destroy_infrastructure.yaml

```diff
@@ -0,0 +1,41 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
```

Comment:
Any reason for the destroy not to use the infrastructure build image?

Reply:
No I don't think so - I'll update this